lib/data_magic/query_builder.rb

```diff
@@ -26,6 +30,21 @@ def generate_squery(params, options, config)
         search_fields_and_ranges(squery, params, config)
       end
 
+      def add_aggregations(params, options, config)
+        # Wrapper for Stretchy aggregation clause builder (which wraps ElasticSearch (ES) :aggs parameter)
```

Comment:
Our current style in the project so far is to put method documentation above the method declaration. Can you move these comment lines above the `def`?